src/App.vue

```diff
@@ -384,6 +392,10 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
         // fetch draft filing
         let draftFiling = await this.fetchDraft()
 
+        // If there is an existing filing, check if it is in a valid state to be edited
+        if (draftFiling) {
+          this.invalidIncorporationApplicationDialog = this.hasInvalidFilingState(draftFiling)
+        }
```

Comment:
If this is not a valid state, should you throw an error to skip the rest of this code block?

Reply:
Also need to clear this flag in resetFlags() below.

Reply:
Also need to check this flag in isErrorDialog() above.